void main(){
  throwError();
}

void throwError() {
  throw Exception("This is an error");
}